### PR TITLE
fix: various fixes

### PR DESCRIPTION
**Pull request by pwltr**
### Description

- fix not subscribed to address after disconnected from electrum server
- fix some drop shadows, ignore warnings
- fix warning for react-navigation non-serializable values passed as props
- Settings: more pixel perfection for settings

### Linked Issues/Tasks

https://app.asana.com/0/0/1204520024497717/f

### Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Refactoring (improving code without creating new functionality)

### Tests

- [ ] Detox test
- [ ] Unit test
- [x] No test

### CODE REVIEW

File: src/utils/wallet/electrum.ts [hunk 1]
```diff
@@ -257,48 +256,51 @@ export const subscribeToAddresses = async ({
...
+		}
+	});
+
+	try {
+		await Promise.all(promises);
+	} catch (e) {
+		console.log(e);
+		return err(e);
+	}
+
```
Comment (pwltr) [inline src/utils/wallet/electrum.ts:303]: just refactoring here and avoiding swallowing errors

File: src/utils/wallet/electrum.ts [hunk 2]
```diff
@@ -756,34 +752,22 @@ export const connectToElectrum = async ({
...
-		startResponse = await electrum.start({
-			network: selectedNetwork,
-			customPeers,
-			net,
-			tls: _tls,
-		});
-		if (!startResponse.error) {
-			break;
-		}
-	}
```
Comment (pwltr) [inline src/utils/wallet/electrum.ts:770]: this retry logic was broken, also we are reconnecting to electrum automatically so it's not needed

File: src/assets/icons/wallet.ts [hunk 3]
```diff
@@ -6,20 +6,30 @@ export const transferIcon = (
...
+</svg>
+`;
 
 export const unitSatoshiIcon = (
 	color = 'white',
 ): string => `<svg width="32" height="32" viewBox="0 0 32 32" xmlns="http://www.w3.org/2000/svg">
-<path d="M0 16C0 7.16344 7.16344 0 16 0C24.8366 0 32 7.16344 32 16C32 24.8366 24.8366 32 16 32C7.16344 32 0 24.8366 0 16Z" fill="${color}" fill-opacity="0.08"/>
+<rect fill="#fff" fill-opacity=".08" height="32" rx="16" width="32"/>
```
Comment (limpbrains) [inline src/assets/icons/wallet.ts:18]: Shouldn't this `fill="#fff"` be also replaced with `fill="${color}"` ?

I usually runs svg through ImageOptim, it squashes SVG in nice one-line strings

File: src/components/Button.tsx [hunk 4]
```diff
@@ -109,11 +109,12 @@ const styles = StyleSheet.create({
...
-		shadowOpacity: 0.8,
-		elevation: 6,
-		shadowRadius: 15,
-		shadowOffset: { width: 1, height: 13 },
+		// TODO: shadows are not drawn correctly on transparent buttons
+		// shadowColor: 'rgba(0, 0, 0, 0.1)',
+		// shadowOpacity: 0.8,
+		// elevation: 6,
+		// shadowRadius: 15,
+		// shadowOffset: { width: 1, height: 13 },
```
Comment (limpbrains) [inline src/components/Button.tsx:117]: Yeah, noticed that too, nice

**Review by limpbrains — APPROVED** [review]

File: src/assets/icons/wallet.ts (hunk 3 above)
Comment (pwltr) [reply src/assets/icons/wallet.ts:18]: > Shouldn't this fill="#fff" be also replaced with fill="${color}" ?

The rect is for the circled background, color should be white transparent like for the other circled icons

**Pull request merged**